lisa/sut_orchestrator/baremetal/cluster/rackmanager.py

```diff
@@ -70,3 +70,9 @@ def reset(self, operation: str) -> None:
             rm_node.execute(f"set system {operation} -i {client.management_port}")
 
         self._log.debug(f"client has been {operation} successfully")
+
+    def get_cluster_capabilities(self) -> ClusterCapabilities:
+        cluster_capabilities = ClusterCapabilities()
+        cluster_capabilities.core_count = 8
```

Comment:
For rack manager, we can get it by two ways.

1. write values in runbook
2. Load real values, when environment is booted up.

Reply:
if use 2, probably some cases will be skipped before deployment?

Reply:
That's ok, we have best tries. But we can still leave default value here, maybe a max value to prevent any case is blocked.